warehouse/templates/base.html

```diff
@@ -141,6 +150,15 @@
         <span class="notification-bar__message">You are using TestPyPI – a separate instance of the Python Package Index that allows you to try distribution tools and processes without affecting the real index.</span>
       </div>
       {% endif %}
+      <noscript>
+      <div class="notification-bar notification-bar--danger">
+        <span class="notification-bar__icon">
+          <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+          <span class="sr-only">Warning:</span>
+        </span>
+        <span class="notification-bar__message">Some features may not work without JavaScript. Please try enabling it if you encounter problems.</span>
+      </div>
+      </noscript>
```

Comment:
If we add this, can you add a TODO comment to remove it once https://github.com/pypa/warehouse/issues/3828 is resolved (and leave a comment on that issue about it)? Everything _should_ work without JavaScript enabled.

Reply:
Will do. Most things work already, but I suspect there are some more subtle bits that might take a bit longer.

Reply:
Indeed, I left a comment on https://github.com/pypa/warehouse/issues/3828 about one in particular.